dataset/dictionary/characters.json5

```
@@ -549,8 +549,8 @@
     ja: "アンデシュドッテル",
     zhCN: "安德斯多特",
     tags: [ "mondstadt", "character-sub" ],
-    notes: "魔女会・コードM。イノシシプリンセスの作者。",
-    notesZh: "魔女会代号M。《野猪公主》的作者。",
+    notes: "魔女会・コードM。イノシシプリンセスの作者。", // TODO: v4.8 期間限定イベントのクエストムービーより、アンヤ・M・アンデシュドッテルがフルネームだと思われる
+    notesZh: "魔女会代号M。《野猪公主》的作者。", // TODO: 从v4.8限时活动任务电影来看，安雅·M·安德斯多特似乎是全名
```

```suggestion
    notes: "魔女会・コードM。イノシシプリンセスの作者。フルネームは「アンヤ・M・アンデシュドッテル」(英: Anya M. Andersdotter, 中: 安雅·M·安德斯多特) と思われる", // Source of her fullname: Event movie in Simulanka in v4.8
    notesZh: "魔女会代号M。《野猪公主》的作者。，安雅·M·安德斯多特 (英: Anya M. Andersdotter, 日: アンヤ・M・アンデシュドッテル) 似乎是全名",
```

Let's add her fullname in the notes.